docs/source/alfacase_definitions/PhysicsDescription.txt

```diff
@@ -16,13 +16,13 @@
             emulsion_model_enabled: bool = True
             emulsion_relative_viscosity_model: \ :class:`EmulsionRelativeViscosityModelType <alfasim_sdk._internal.constants.EmulsionRelativeViscosityModelType>`\  = EmulsionRelativeViscosityModelType.Brinkman1952
             emulsion_pal_rhodes_phi_rel_100: \ :class:`Optional <typing.Optional>`\[Scalar] = Scalar(0.765, '-', 'dimensionless')
-            emulsion_woelflin_a: \ :class:`Scalar <barril.units.Scalar>`\  = Scalar(4.2, '-', 'dimensionless')
-            emulsion_woelflin_b: \ :class:`Scalar <barril.units.Scalar>`\  = Scalar(2.5, '-', 'dimensionless')
+            emulsion_woelflin_a: \ :class:`Optional <typing.Optional>`\[Scalar] = Scalar(4.2, '-', 'dimensionless')
```

Comment:
Strange, why did this became "Optional"?

Reply:
> Untrue. I forgot to handle the case with Scalar and ScalarExpression type in the generation docs functions.

I was going to point out that it is being tagged as "Optional" because prior to "unions" we only had `typing.Optional[foo]` that is a syntactic sugar for `typing.Union[foo, None]` them all `Scalar | ScalarExpression` where assumed to be just `typing.Optional[Scalar]`.